hasher-matcher-actioner/webapp/src/Api.tsx

```diff
@@ -324,23 +324,23 @@ export async function fetchHashCount(): Promise<Response> {
 
 // TODO remove the trailing slash from the API URL, then add back the leading slash for /actions/ and /action-rules/ endpoints.
 
+// This class should be kept in sync with python class ActionPerformer
+export type APIActionPerformer = {
+  name: string;
+  config_subtype: string;
+  url: string;
+  headers: string;
+};
+
 type AllActions = {
   error_message: string;
-  actions_response: Array<{
-    name: string;
-    config_subtype: string;
-    url: string;
-    headers: string;
-  }>;
+  actions_response: Array<APIActionPerformer>;
 };
 
 export async function fetchAllActions(): Promise<Action[]> {
   return apiGet<AllActions>('actions/').then(response => {
     if (response && !response.error_message && response.actions_response) {
-      return response.actions_response.map(
-        ({name, config_subtype, url, headers}) =>
-          new Action(name, config_subtype, url, headers),
-      );
+      return response.actions_response.map(action => new Action(action));
```

Comment:
did you mean for this do be `APIActionPerformer` if not why does it exist compared to `Action`

Reply:
Moved `APIActionPerformer` (now called `BackendActionPerformer`) so that it is only referenced in this file which does all the translation `Action <-> BackendActionPerformer`